src/components/pages/AllCollections.tsx

```diff
@@ -41,9 +42,17 @@ const AllCollections: React.FC<AllCollectionsProps> = () => {
 
   useEffect(() => {
     const { query } = router;
-    if (query && query[UrlSearch.KeywordSearch]) {
-      const keywordSearch = query[UrlSearch.KeywordSearch] as string;
-      setSearchKeywords(keywordSearch);
+    if (query) {
+      if (query[UrlSearch.KeywordSearch]) {
+        const keywordSearch = query[UrlSearch.KeywordSearch] as string;
```

Comment:
Info: why is the cast required here?

Reply:
Because the return is a `string | string[]` for example when you specify an array of values with the repeat format. In my case I should get a string. I could check for it though.